lib/pure/concurrency/threadpool.nim

```diff
@@ -250,11 +252,14 @@ proc `^`*[T](fv: FlowVar[T]): T =
     result = fv.blob
 
 proc blockUntilAny*(flowVars: openArray[FlowVarBase]): int =
-  ## awaits any of the given flowVars. Returns the index of one flowVar for
-  ## which a value arrived. A flowVar only supports one call to 'blockUntilAny' at
-  ## the same time. That means if you blockUntilAny([a,b]) and blockUntilAny([b,c]) the second
+  ## Awaits any of the given flowVars. Returns the index of one flowVar for
+  ## which a value arrived.
+  ##
+  ## A flowVar only supports one call to 'blockUntilAny' at the same time.
+  ## That means if you `blockUntilAny([a,b])` and `blockUntilAny([b,c])` the second
```

Comment:
I tested this (while working on `logging`) and both single and double backticks appear to do the same thing, which is to render text as monospace. According to this [reST reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup), the meaning of single backticks is up to the application, and if I'm understanding [this bit of code correctly](https://github.com/nim-lang/Nim/blob/devel/lib/packages/docutils/rstgen.nim#L1141), it currently doesn't matter whether single or double backticks are used, as the same HTML is written in both cases.

Reply:
> AFAIK you need two backticks here, this will just make it italic which isn't probably what you want.

You haven't been following recent development, have you? ;)

Single backticks (markdown) are now the same as double backticks (rst), and markdown syntax for links also works.